.changeset/flat-actors-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/link": patch
```

Comment:
It is "@yamada-ui/next", not `"@yamada-ui/link"`.

Reply:
fixed
[0b5592f](https://github.com/yamada-ui/yamada-ui/pull/3034/commits/0b5592f5037d778a1a608a9c991acdfaa1f7bf41)